pipelines/h5c/rtp/v1/stage_1_task_scripts/do_LIBRARIAN_REDCAL_KNOWN_GOOD.sh

```diff
@@ -0,0 +1,51 @@
+#! /bin/bash
+set -e
+
+# This function uploads all redcal_known_good files to the Librarian
+
+# import common functions
+src_dir="$(dirname "$0")"
+source ${src_dir}/_common.sh
+
+# Parameters are set in the configuration file. Here we define their positions,
+# which must be consistent with the config.
+# 1 - filename
+# 2 - extension: Extension to be appended to the file name.
+# 3 - upload_to_librarian: global boolean trigger
+# 4 - librarian_redcal_known_good: boolean trigger for this step
+fn="${1}"
+upload_to_librarian="${2}"
+librarian_ant_metrics="${3}"
+
+bn=`basename ${fn}`
+jd=$(get_int_jd ${fn})
+
+# upload results to librarian if desired
+if [ "${upload_to_librarian}" == "True" ]; then
+    if [ "${librarian_redcal_known_good}" == "True" ]; then
+
+        # Compress all redcal_known_good files into one with a JD corresponding to $fn        
```

Comment:
I'd clarify that it's one compressed file per output file type (which I think is a fine choice).

Reply:
Done